vtexdocs/devportal
# Feature/e2e-tests

#### What is the purpose of this pull request?

Add more robust End-to-End tests and generate a log file that will later be used to report errors in the PR.


#### How should this be manually tested?
Go to the cypress action result  (click on details) and check if the result box is like this
![image](https://user-images.githubusercontent.com/42784961/226666774-e13db6cd-b8e2-4d02-8ffc-a09288c25a9f.png)

#### Types of changes
